ibm_catalog.json

```
@@ -0,0 +1,115 @@
+{
+    "products": [
+        {
+            "features": [
+                {
+                    "description": "Multifactor authentication (None - Federated Users - All - Email/TOPT/U2F based)",
+                    "title": "Multifactor authentication"
+                },
+                {
+                    "description": "Restrict API key creation (on-off)\nRestrict service ID creation (on-off)",
+                    "title": "Restrict API Key and Service IDs"
+                },
+                {
+                    "description": "Session activity timeout (seconds)\nSession inactivity timeout (seconds)",
+                    "title": "Session timeouts"
+                },
+                {
+                    "description": "Access token expiration (seconds)\nRefresh token expiration (seconds)",
+                    "title": "Token Expiration"
+                },
+                {
+                    "description": "off/Any Allowed - on/Allow only specified IP subnets or IP addresses. \n\nTwo control modes are supported Monitor \u0026 Restrict",
+                    "title": "Restrict IP address access"
+                },
+                {
+                    "description": "Global shell settings (on-off)\nPublic access group (on-off)",
+                    "title": "Miscellaneuous"
+                }
+            ],
+            "flavors": [
+                {
+                    "architecture": {},
+                    "compliance": {},
+                    "configuration": [],
+                    "install_type": "fullstack",
+                    "outputs": [
+                        {
+                            "description": "Current active session timeout",
+                            "key": "account_iam_active_session_timeout"
+                        },
+                        {
+                            "description": "Current state of public access group setting",
+                            "key": "account_public_access"
+                        },
+                        {
+                            "description": "Current refresh token expiration",
+                            "key": "account_iam_refresh_token_expiration"
+                        },
+                        {
+                            "description": "Current allowed IP addresses enforcement control mode, will indicate RESTRICT if account_allowed_ip_addresses_enforced is TRUE",
```

```suggestion
                            "description": "Enforcement mode for allowed IP addresses. RESTRICT is returned if account_allowed_ip_addresses_enforced is set to true. Otherwise, XXX is returned.",
```

I'm not the target audience, but I have a hard time understanding these outputs. Here's a suggestion for how I interpreted this output. I added a placeholder for a 2nd value. We should document all values that can be returned. If the output is not in all CAPS, we can use double quotation marks for the value (for ex, `"Restrict" is returned if...`)

---

Actually, I took the description from the readme of this module, but for a few features, I need to make it short because it has a max 140 char limit. It also has a limit to have max 6 features at the offering level. So I will add some features to the "flavor" level. As your suggested change is 148 chars, I think I will get rid of `Otherwise, XXX is returned.`